src/Makefile.3ds

```diff
@@ -200,6 +203,8 @@ endif
 ifneq ($(DEPSDIR),$(BUILD))
 $(DEPSDIR):
```

Comment:
My suggestion was wrong here for multiple reasons.  This works in my hands, replacing lines 192 through 210:

```
all: $(BUILD) $(BUILD)/borg $(GFXBUILD) $(DEPSDIR) $(DEPSDIR)/borg $(ROMFS_T3XFILES) $(T3XHFILES)
        @$(MAKE) --no-print-directory -C $(BUILD) -f $(CURDIR)/Makefile.3ds $(PROGNAME).3dsx

$(BUILD) $(BUILD)/borg:
        @mkdir -p $@

ifneq ($(GFXBUILD),$(BUILD))
$(GFXBUILD):
        @mkdir -p $@
endif

ifneq ($(DEPSDIR),$(BUILD))     
$(DEPSDIR) $(DEPSDIR)/borg:
        @mkdir -p $@
endif
```